src/dict/dicts.js

```diff
@@ -79,7 +79,8 @@ export default {
     network: {
       footerText: "",
       headerText: "",
-      smallSocialButtonsHeader: "Login with"
+      smallSocialButtonsHeader: "Login with",
+      bigSocialButtonsPrefix: "Login with"
```

Comment:
Can you change the `bigSocialButtonsPrefix` to `socialButtonLabel` and include a `provider` placeholder for the identity provider? For an example of a placeholder see the [welcome translation](https://github.com/auth0/lock-passwordless/blob/df183e846e76b45ce874f87eb6721ca575da21f0/src/dict/dicts.js#L152)